### ISSUE TITLE
Feature Request: Search Google for xyz in new container

### ISSUE THREAD

**Issue by Genius1237** [stoically/temporary-containers#564]
When one right clicks on a link, there is an option to open the new tab in a new container. Would it be possible to add the same feature in for the "Search Google for xyz" that pops up when you highlight some text and right click on it?

**Comment by stoically** [comment]
Possible yes, tho I'd prefer the search engine to be configurable with no default set.

**Comment by Genius1237** [comment]
Yes. Whatever is the default search engine is set in the browser.

**Comment by stoically** [comment]
Not sure an API to get that information exists, but that would be a possibility, yeah

**Comment by Genius1237** [comment]
Firefox already has a "Search with Google" (replace google with default search engine) option that comes up when you right click on some selected text. Would it be possible to just hook into what this does to implement this feature?

**Comment by stoically** [comment]
Afaik there's no hook or API for the context menu to hook into an already existing entry. Only new entries can be added.
